src/Service/Resolvers/Sql Query Structures/SqlUpsertQueryStructure.cs

```diff
@@ -72,7 +72,7 @@ public class SqlUpsertQueryStructure : BaseSqlQueryStructure
               authorizationResolver: authorizationResolver,
               gQLFilterParser: gQLFilterParser,
               entityName: entityName,
-              operationType: Config.Operation.Update,
+              operationType: Config.Operation.Upsert,
```

Comment:
> operationType: Config.Operation.Upsert, [](http://example.com/codeflow?start=13&length=40)

What was the effect of this change? Was `Update` causing some undesired behavior? Given the name of the query structure, I would agree that it should be Upsert.

Reply:
Because previously we were supporting db policies only for update part of an upsert operation, we were passing only `Operation.Update` here. Now since an Upsert operation supports both insert and update db policy, we are passing `Operation.Upsert` here. It is to be noted here that this `operationType` parameter is only used to determine corresponding to what operation the db policy is to be added.